packages/react-components/react-button/stories/Button/ButtonLoading.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { buttonClassNames, makeStyles, shorthands, tokens, Button, Spinner } from '@fluentui/react-components';
+import { CheckmarkFilled } from '@fluentui/react-icons';
+import { useTimeout } from '@fluentui/react-utilities';
+
+const useStyles = makeStyles({
+  wrapper: {
+    columnGap: '15px',
+    display: 'flex',
+  },
+  buttonNonInteractive: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+    color: tokens.colorNeutralForeground1,
+    cursor: 'default',
+    pointerEvents: 'none',
+
+    [`& .${buttonClassNames.icon}`]: {
```

Comment:
Super nit, feel free to disregard. That said, this style increase specificity on the `icon` slot. Since we are already tracking the button's state via JS you can conditionally apply the style to avoid increasing specificity.

Reply:
I need to do it this way specifically because this is already being done on the button styles, so I can override it only by doing it this way.